mods/stairs/init.lua

```diff
@@ -550,6 +651,16 @@ stairs.register_stair_and_slab(
 )
 
 stairs.register_stair_and_slab(
+	"clay",
+	"default:clay",
+	{cracky = 3},
+	{"default_clay.png"},
+	"Clay Stair",
+	"Clay Slab",
+	default.node_sound_stone_defaults()
+)
```

Comment:
I disagree with adding new stair types in the same pull request. Make Clay and Diamond Block Stairs and Slabs be a different pull request.